rest/taskana-rest-spring-example/src/test/java/pro/taskana/rest/ClassificationControllerIntTest.java

```diff
@@ -273,9 +279,31 @@ public void testUpdateClassificationPrioServiceLevel()
         assertEquals(200, con.getResponseCode());
         con.disconnect();
 
-        // wait a minute to give JobScheduler a chance to run
-        LOGGER.info("About to sleep for 70 seconds to give JobScheduler a chance to process the classification change");
-        Thread.sleep(70000);
+        // wait taskana.jobscheduler.delay + 5 seconds to give JobScheduler a chance to run
+        String cron = env.getProperty("taskana.jobscheduler.cron");
+        CronTrigger trigger = new CronTrigger(cron);
+        TriggerContext context = new TriggerContext() {
+
+            @Override
+            public Date lastScheduledExecutionTime() {
+                return null;
+            }
+
+            @Override
+            public Date lastActualExecutionTime() {
+                return null;
+            }
+
+            @Override
+            public Date lastCompletionTime() {
+                return null;
+            }
+        };
+        long delay = trigger.nextExecutionTime(context).getTime() - (new Date()).getTime() + 2000;
+
+        LOGGER.info("About to sleep for " + delay / 1000
```

Comment:
This is not working anymore, is it?

Reply:
Yes, it is working. It parses the cron expression and uses the result to calculate the delay up to the next run of JobScheduler ...

Reply:
I have noticed that rest tests are sleeping for sometime(1,5min) to process the job execution.
I suggest to make cron configurable this way test could configure job schedule to less time(5 sec?) and tests will run faster.

Reply:
This is exactly one test: ClassificationControllerIntTest.testUpdateClassificationPrioServiceLevel().
With this pull request, this test parses the cron expression that controls JobScheduler and calculates the delay between now and the next trigger time. Then it sleeps for 2 seconds more that that delay...

Reply:
ok, perfect.